[2024-12-10 17:28:51,650 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 17:31:44,659 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 17:32:39,809 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 17:33:12,923 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 17:37:57,400 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 17:38:07,268 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:22:16,964 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:23:06,771 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:23:31,366 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:26:46,244 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:29:33,948 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:30:37,420 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-10 22:32:02,098 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-11 12:23:33,141 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie
[2024-12-12 10:55:53,405 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 10:57:24,051 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 10:58:14,535 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:02:12,584 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:02:18,026 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:02:56,906 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:05:58,724 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:06:06,393 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:16:21,995 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:16:59,760 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:18:47,386 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 11:18:59,387 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-12 14:17:19,163 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-12 14:20:19,736 - INFO]__main__: azerty_box a utilisé le /reset dans random
[2024-12-12 14:20:26,348 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Roupilion
[2024-12-12 14:20:30,772 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko
[2024-12-12 14:20:32,730 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu SuperfouÃ©tar
[2024-12-12 14:20:35,893 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu trÃ©oigno
[2024-12-12 14:20:38,197 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu SuperfouÃ©tar
[2024-12-12 14:20:41,094 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Lamlam'mou
[2024-12-12 14:20:43,246 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ronimpec
[2024-12-12 14:20:47,207 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi
[2024-12-12 14:20:49,934 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko
[2024-12-12 14:20:51,902 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi
[2024-12-12 14:20:53,532 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ronimpec
[2024-12-12 14:20:56,050 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi
[2024-12-12 14:20:57,899 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie
[2024-12-12 14:21:00,734 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu fouÃ©tar
[2024-12-12 14:21:03,510 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samoussrai
[2024-12-12 14:21:05,300 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samoussrai
[2024-12-12 14:21:08,133 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-13 16:40:07,005 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-13 16:40:46,682 - INFO]__main__: azerty_box a utilisé le /help dans random
[2024-12-17 19:32:43,587 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Lescampette, de rang D
[2024-12-17 19:32:45,822 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang D
[2024-12-17 19:32:46,639 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Heaumer, de rang E
[2024-12-17 19:32:49,347 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Eddylapid, de rang A
[2024-12-17 19:32:51,758 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samuren, de rang E
[2024-12-17 19:32:54,240 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang E
[2024-12-17 19:32:56,543 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ralbouc, de rang C
[2024-12-17 19:32:58,896 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-17 19:33:01,353 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‰pigerien, de rang E
[2024-12-17 19:33:03,799 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu KÅ“ud, de rang D
[2024-12-17 19:33:06,481 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang E
[2024-12-17 19:33:08,880 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Heaumer, de rang D
[2024-12-17 19:33:11,257 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Shogunyan, de rang S et legendaire
[2024-12-17 19:33:14,692 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Valetino, de rang B
[2024-12-17 19:33:17,179 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang D
[2024-12-17 19:33:19,753 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Violette, de rang C
[2024-12-17 19:33:22,268 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu MoiYihin, de rang C
[2024-12-17 19:33:24,618 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pagno, de rang C
[2024-12-17 19:33:27,246 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang E
[2024-12-17 19:33:29,693 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang E
[2024-12-17 19:33:31,275 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Brumette, de rang D
[2024-12-17 19:33:33,881 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang E
[2024-12-17 19:33:37,130 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-17 19:34:36,123 - INFO]__main__: azerty_box a utilisé le /reset dans random
[2024-12-17 19:34:39,456 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Eddylapid, de rang A
[2024-12-17 19:34:40,489 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‰pigerien, de rang E
[2024-12-17 19:34:42,999 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu BarrakÃ©da, de rang B
[2024-12-17 19:34:45,593 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Enma Solaire, de rang S et divinite / Enma
[2024-12-17 19:34:48,008 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang E
[2024-12-17 19:34:50,437 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang E
[2024-12-17 19:34:52,832 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‰pigerien, de rang E
[2024-12-17 19:34:55,268 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sushiyama, de rang B
[2024-12-17 19:34:58,913 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang D
[2024-12-17 19:35:00,408 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bouh, de rang C
[2024-12-17 19:35:02,836 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Coach Antonic, de rang C
[2024-12-17 19:35:05,243 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Enma Solaire, de rang S et divinite / Enma
[2024-12-17 19:35:07,673 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Holy Rabby, de rang S
[2024-12-17 19:35:10,241 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu El Galardo, de rang D
[2024-12-17 19:35:12,615 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ToutouÃ¯e, de rang D
[2024-12-17 19:35:20,485 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-17 19:35:26,603 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Heaumer, de rang D
[2024-12-17 19:35:27,465 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2024-12-17 19:35:29,902 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mochimacho, de rang E
[2024-12-17 19:35:32,550 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang E
[2024-12-17 19:35:35,049 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Marra, de rang D
[2024-12-17 19:35:37,522 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pagno, de rang C
[2024-12-17 19:35:39,969 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-17 19:35:42,400 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ToutouÃ¯e, de rang D
[2024-12-17 19:35:45,240 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sabri, de rang A
[2024-12-17 19:35:46,699 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang E
[2024-12-17 19:35:49,167 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang E
[2024-12-17 19:35:51,586 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mochimacho, de rang E
[2024-12-17 19:35:55,090 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang E
[2024-12-17 19:35:57,546 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hovernyan, de rang B
[2024-12-17 19:35:59,969 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang E
[2024-12-17 19:36:02,442 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bonneto, de rang C
[2024-12-17 19:36:04,806 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ultramax K, de rang B
[2024-12-17 19:36:07,246 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mochimacho, de rang E
[2024-12-17 19:36:09,796 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang E
[2024-12-17 19:36:12,192 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu T. Energison, de rang S
[2024-12-17 19:36:14,635 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sale de bain, de rang E
[2024-12-17 19:36:17,041 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Feulion, de rang C
[2024-12-17 19:36:19,536 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu MoiYihin, de rang C
[2024-12-17 19:36:22,016 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu El Galardo, de rang E
[2024-12-17 19:36:24,419 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang E
[2024-12-17 19:36:26,833 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang E
[2024-12-17 19:36:29,293 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang E
[2024-12-17 19:36:31,790 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ultramax K, de rang B
[2024-12-17 19:36:34,174 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang E
[2024-12-17 19:36:36,627 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sale de bain, de rang E
[2024-12-17 19:36:41,790 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2024-12-17 19:36:44,275 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Scatcheur, de rang B
[2024-12-17 19:36:46,667 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-17 19:36:49,056 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Barbokai, de rang C
[2024-12-17 19:36:52,433 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mochimacho, de rang E
[2024-12-17 19:36:54,866 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bon-huiii, de rang E
[2024-12-17 19:36:57,523 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‰pigerien, de rang D
[2024-12-17 19:37:00,329 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang E
[2024-12-17 19:37:01,774 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang E
[2024-12-17 19:37:04,165 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bonneto, de rang C
[2024-12-17 19:37:06,680 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Heaumer, de rang E
[2024-12-17 19:37:09,372 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Enma 2.0, de rang S et divinite / Enma
[2024-12-17 19:37:11,822 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Heaumer, de rang D
[2024-12-17 19:37:14,293 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mochimacho, de rang E
[2024-12-17 19:37:16,711 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu El Galardo, de rang E
[2024-12-17 20:47:20,600 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-17 20:47:32,276 - INFO]__main__: azerty_box a utilisé le /reset dans random
[2024-12-17 20:47:35,683 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Coach Antonic, de rang C
[2024-12-17 20:47:36,596 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Babalance, de rang B
[2024-12-17 20:47:39,079 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang D
[2024-12-17 20:47:41,834 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gastong, de rang E
[2024-12-17 20:47:43,741 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2024-12-17 20:47:46,285 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Shogunyan, de rang S et legendaire
[2024-12-17 20:47:48,932 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-17 20:47:51,365 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Triptic-tac, de rang A
[2024-12-17 20:47:53,768 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ApÃ©licain, de rang C
[2024-12-17 20:47:56,197 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Steuf, de rang C
[2024-12-17 20:47:58,611 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ApÃ©licain, de rang C
[2024-12-17 20:48:01,033 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hovernyan, de rang B
[2024-12-17 20:48:03,443 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-17 20:48:06,876 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Kapunki, de rang A
[2024-12-17 20:48:08,566 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Wattson, de rang B
[2024-12-17 20:48:10,998 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko, de rang E
[2024-12-17 20:48:13,591 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Steuf, de rang C
[2024-12-17 20:48:16,007 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hotei, de rang S
[2024-12-17 20:48:18,445 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ToutouÃ¯e, de rang D
[2024-12-17 20:48:21,104 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Steuf, de rang C
[2024-12-17 20:48:23,590 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu trÃ©pigno, de rang E
[2024-12-17 20:48:25,974 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Matchou, de rang D
[2024-12-17 20:48:28,499 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hibourling, de rang B
[2024-12-17 20:48:30,906 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Matchou, de rang D
[2024-12-17 20:48:34,641 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hibrouille, de rang C
[2024-12-17 20:48:36,133 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Padelou, de rang A
[2024-12-17 20:48:38,554 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hibourling, de rang B
[2024-12-17 20:48:40,911 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ChaipÃ´, de rang E
[2024-12-17 20:48:44,316 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gastong, de rang E
[2024-12-17 20:48:46,801 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Duc Delafeuille, de rang E
[2024-12-17 20:48:49,316 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Typ, de rang S (issue d'une collaboration)
[2024-12-17 20:48:52,199 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko, de rang E
[2024-12-17 20:48:53,718 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gagalurin, de rang E
[2024-12-17 20:48:56,341 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu El Galardo, de rang D
[2024-12-17 20:48:58,849 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bastata, de rang C
[2024-12-17 20:49:01,315 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-17 20:49:03,819 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‚miral, de rang S
[2024-12-17 20:49:06,236 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gagalurin, de rang E
[2024-12-17 20:49:08,598 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang D
[2024-12-17 20:49:10,993 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2024-12-17 20:49:13,403 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mythovni, de rang C
[2024-12-17 20:49:16,906 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-17 20:49:19,290 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Vibrilion, de rang B
[2024-12-17 20:49:21,705 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2024-12-17 20:49:24,180 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Roupilion, de rang E
[2024-12-17 20:49:26,598 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu KÅ“ud, de rang D
[2024-12-17 20:49:29,231 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang D
[2024-12-17 20:49:31,832 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ToutouÃ¯e, de rang D
[2024-12-17 20:49:34,237 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-17 20:49:36,748 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samuren, de rang D
[2024-12-17 20:49:39,206 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-17 20:49:41,701 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Corailline, de rang S
[2024-12-17 20:49:44,101 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bobdruche, de rang B
[2024-12-17 20:49:46,529 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Donjouant, de rang A
[2024-12-17 20:49:48,986 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sale de bain, de rang D
[2024-12-17 20:49:53,262 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-17 20:49:53,571 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gagalurin, de rang E
[2024-12-17 20:49:56,325 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2024-12-17 20:49:58,796 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Curryogi, de rang A
[2024-12-17 20:50:01,471 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Shurikenny, de rang S
[2024-12-17 20:50:03,925 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gagalurin, de rang E
[2024-12-17 20:50:06,842 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-17 20:50:09,387 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-17 20:50:11,836 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang D
[2024-12-17 20:50:14,371 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-17 20:50:17,089 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2024-12-17 20:50:18,516 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hovernyan, de rang B
[2024-12-17 20:50:20,870 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S (issue d'une collaboration)
[2024-12-17 20:50:24,301 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2024-12-17 20:50:26,738 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pleurapluie, de rang E
[2024-12-17 20:50:29,135 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu KÅ“ud, de rang D
[2024-12-17 20:50:31,568 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-17 20:50:34,088 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2024-12-17 20:50:36,714 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2024-12-17 20:50:39,235 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sumochi, de rang D
[2024-12-17 20:50:41,731 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2024-12-17 20:50:44,147 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bouh, de rang C
[2024-12-17 20:50:46,556 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Feuildechou, de rang A
[2024-12-17 20:50:48,951 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-17 20:50:51,421 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Rugis, de rang A
[2024-12-17 20:50:53,850 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pleurapluie, de rang E
[2024-12-17 20:51:09,118 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:22:02,546 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Duc Delafeuille, de rang E
[2024-12-18 13:23:52,274 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samoussrai, de rang E
[2024-12-18 13:23:55,337 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-18 13:23:56,372 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Narinos, de rang D
[2024-12-18 13:23:58,842 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-18 13:24:01,267 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-18 13:24:03,755 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Moinon, de rang C
[2024-12-18 13:24:06,184 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samoussrai, de rang E
[2024-12-18 13:24:08,740 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samuren, de rang D
[2024-12-18 13:24:11,165 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu BarrakÃ©da, de rang B
[2024-12-18 13:24:13,715 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Lescampette, de rang D
[2024-12-18 13:24:16,154 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Octorbot, de rang S
[2024-12-18 13:24:18,661 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ChaipÃ´, de rang E
[2024-12-18 13:24:21,200 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CalimÃ©balle, de rang D
[2024-12-18 13:24:23,684 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-18 13:24:26,071 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ChaipÃ´, de rang E
[2024-12-18 13:24:28,488 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Brumette, de rang D
[2024-12-18 13:24:40,813 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:25:41,387 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:26:40,700 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:27:22,449 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:28:00,672 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:30:30,020 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:32:25,335 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:34:00,015 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-18 13:34:11,758 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu MÃ©mogre, de rang B
[2024-12-18 13:34:12,678 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang D
[2024-12-18 13:34:15,419 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu El Galardo, de rang D
[2024-12-18 13:34:16,986 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Marra, de rang D
[2024-12-18 13:34:19,579 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko, de rang E
[2024-12-18 13:34:23,055 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Imadjinn, de rang B
[2024-12-18 13:34:24,699 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Woko, de rang E
[2024-12-18 13:34:27,182 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Steuf, de rang C
[2024-12-18 13:34:29,760 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Volroulo, de rang B
[2024-12-18 13:34:32,213 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Toutournemal, de rang E
[2024-12-18 13:34:34,736 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ChaipÃ´, de rang E
[2024-12-18 13:34:37,167 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu KÅ“ud, de rang D
[2024-12-18 13:34:39,543 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Donjouant, de rang A
[2024-12-18 13:34:41,995 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Mythovni, de rang C
[2024-12-18 13:34:45,401 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2024-12-18 13:34:48,131 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hovernyan, de rang B
[2024-12-18 13:34:49,710 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pleurapluie, de rang E
[2024-12-18 13:34:52,175 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-18 13:34:54,550 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-18 13:34:56,966 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2024-12-18 13:35:00,407 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gagalurin, de rang E
[2024-12-18 13:35:02,938 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bobdruche, de rang B
[2024-12-18 13:35:05,344 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2024-12-18 13:35:07,768 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu trÃ©pigno, de rang E
[2024-12-18 13:35:10,201 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-18 13:35:12,595 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Tengu, de rang S
[2024-12-18 13:35:14,987 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Lionguiste, de rang C
[2024-12-18 13:35:27,425 - INFO]__main__: azerty_box a utilisé le /inventaire dans random
[2024-12-23 11:00:42,985 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Coach Antonic, de rang C
[2024-12-23 11:00:44,130 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2024-12-23 11:00:46,306 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-23 11:00:49,035 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gastong, de rang E
[2024-12-23 11:00:51,559 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Toumou, de rang E
[2024-12-23 11:00:54,146 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Espi, de rang B
[2024-12-23 11:00:55,617 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2024-12-23 11:00:59,135 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samuren, de rang D
[2024-12-23 11:01:01,646 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu ChaipÃ´, de rang E
[2024-12-23 11:01:44,521 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Marra, de rang D
[2024-12-23 11:01:46,302 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hibrouille, de rang C
[2024-12-23 11:01:53,463 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2024-12-23 11:01:54,575 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Sale de bain, de rang D
[2024-12-23 11:01:56,649 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Professeur Plumage, de rang B
[2024-12-23 11:02:00,406 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Nuiroir, de rang A
[2024-12-23 11:02:03,045 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2024-12-23 11:02:05,429 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-23 11:02:07,077 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ã‰pigerien, de rang D
[2024-12-23 11:02:09,208 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2024-12-23 11:02:11,931 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hiblusion, de rang C
[2024-12-23 11:02:14,473 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu B3-NK1, de rang B
[2024-12-23 11:02:16,966 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Moinon, de rang C
[2024-12-23 11:02:19,405 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2024-12-23 11:02:21,858 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Toutournemal, de rang E
[2024-12-23 11:02:24,237 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2024-12-23 11:02:26,708 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-23 11:02:29,185 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu trÃ©pigno, de rang E
[2024-12-23 11:02:31,900 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Barbokai, de rang C
[2024-12-23 11:02:34,411 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Imadjinn, de rang B
[2024-12-23 11:02:36,969 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Marra, de rang D
[2024-12-23 11:02:39,382 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2024-12-23 11:03:26,588 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2024-12-23 11:03:29,240 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2024-12-23 11:17:30,309 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Matchou, de rang D
[2024-12-23 11:18:00,014 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Toumou, de rang E
[2024-12-23 11:18:38,124 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Toutournemal, de rang E
[2024-12-23 11:19:08,870 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2024-12-26 17:39:06,099 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu MÃ©mÃ©tal, de rang C
[2024-12-27 21:06:15,116 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu AlfoncÃ©o, de rang B
[2025-01-13 19:06:28,245 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Enma 2.0, de rang S et divinite / Enma
[2025-01-13 19:06:35,528 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Cupistol, de rang D
[2025-01-18 16:19:42,897 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ralbouc, de rang C
[2025-01-18 16:19:52,953 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Brumette, de rang D
[2025-01-18 16:19:55,520 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2025-01-18 16:45:22,553 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Agonigiri, de rang D
[2025-01-18 16:45:32,522 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2025-01-18 16:45:35,876 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2025-01-18 16:45:38,642 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Barbokai, de rang C
[2025-01-18 16:45:40,921 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bon-huiii, de rang D
[2025-01-18 16:45:41,259 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2025-01-18 16:45:44,722 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Triptic-tac, de rang A
[2025-01-18 16:45:47,173 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Scatcheur, de rang B
[2025-01-18 16:45:49,624 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2025-01-18 16:45:52,116 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Roupilion, de rang E
[2025-01-18 16:45:54,626 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Matchou, de rang D
[2025-01-18 16:45:57,042 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu trÃ©pigno, de rang E
[2025-01-18 16:45:59,573 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Antitaie, de rang E
[2025-01-18 16:46:02,017 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Vibrilion, de rang B
[2025-01-18 16:46:04,587 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Duc Delafeuille, de rang E
[2025-01-18 16:46:06,979 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Oronyan, de rang S
[2025-01-18 16:46:09,407 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Boulet Frit, de rang D
[2025-01-18 16:46:11,769 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Samoussrai, de rang E
[2025-01-18 16:46:14,191 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Ronimpec, de rang B
[2025-01-18 16:47:31,631 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Asura, de rang S et legendaire
[2025-01-18 16:47:33,963 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Komashura, de rang S et legendaire
[2025-01-18 16:47:35,549 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Gorgouille, de rang S et legendaire
[2025-01-18 16:47:37,293 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Platinos, de rang S et legendaire
[2025-01-18 16:47:39,725 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Komashura, de rang S et legendaire
[2025-01-18 16:47:42,113 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Baratina, de rang S et legendaire
[2025-01-18 16:47:44,560 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Komashura, de rang S et legendaire
[2025-01-18 16:47:47,038 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu CamaÃ¯eul, de rang S et legendaire
[2025-01-18 16:48:18,675 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Pandore, de rang S et trÃ©sor
[2025-01-18 16:50:41,474 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Typ, de rang S spÃ©cial
[2025-01-18 16:51:25,551 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Enma, de rang S et divinite / Enma
[2025-01-18 16:51:30,812 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Zazel le Bon, de rang S et divinite / Enma
[2025-01-18 16:51:32,933 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Zazel le Bon, de rang S et divinite / Enma
[2025-01-18 16:51:34,711 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Hydreux, de rang Boss
[2025-01-18 16:51:36,402 - INFO]__main__: azerty_box a utilisé le /yokai dans random, il a eu Zazel le Bon, de rang S et divinite / Enma
[2025-01-21 21:38:52,354 - ERROR]__main__: 
Error message :
can only concatenate str (not "int") to str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : time.struct_time(tm_year=2025, tm_mon=1, tm_mday=21, tm_hour=21, tm_min=38, tm_sec=51, tm_wday=1, tm_yday=21, tm_isdst=0)
- Commande qui a causé l'erreur : reset
[2025-01-21 21:56:05,654 - ERROR]__main__: 
Error message :
can only concatenate str (not "int") to str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : time.struct_time(tm_year=2025, tm_mon=1, tm_mday=21, tm_hour=21, tm_min=56, tm_sec=5, tm_wday=1, tm_yday=21, tm_isdst=0)
- Commande qui a causé l'erreur : reset
[2025-01-21 21:56:06,441 - ERROR]__main__: 
Error message :
can only concatenate str (not "int") to str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : time.struct_time(tm_year=2025, tm_mon=1, tm_mday=21, tm_hour=21, tm_min=56, tm_sec=5, tm_wday=1, tm_yday=21, tm_isdst=0)
- Commande qui a causé l'erreur : reset
[2025-01-21 21:56:09,189 - ERROR]__main__: 
Error message :
can only concatenate str (not "int") to str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : time.struct_time(tm_year=2025, tm_mon=1, tm_mday=21, tm_hour=21, tm_min=56, tm_sec=8, tm_wday=1, tm_yday=21, tm_isdst=0)
- Commande qui a causé l'erreur : reset
[2025-01-21 21:56:41,185 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bushidos, de rang A
[2025-01-21 21:56:42,344 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Feulion, de rang C
[2025-01-21 21:56:44,595 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Mythovni, de rang C
[2025-01-21 21:56:47,089 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Gastong, de rang E
[2025-01-21 21:56:49,472 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Wattson, de rang B
[2025-01-21 21:56:51,920 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2025-01-21 21:56:54,340 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Robonyan Ultime, de rang S
[2025-01-21 21:56:56,714 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2025-01-21 21:56:59,218 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu MaÃ®tre Oden, de rang A
[2025-01-21 21:57:01,625 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ultramax N, de rang B
[2025-01-21 21:57:04,033 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Sabri, de rang A
[2025-01-21 21:57:06,503 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu trÃ©pigno, de rang E
[2025-01-21 21:57:08,959 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2025-01-21 21:57:12,416 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu CamaÃ¯eul, de rang S et legendaire
[2025-01-21 21:57:14,835 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu CamaÃ¯eul, de rang S et legendaire
[2025-01-21 21:57:17,401 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2025-01-21 21:57:19,819 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Morvobec, de rang E
[2025-01-21 21:58:48,514 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Domniscian 2.0, de rang S et divinite / Enma
[2025-01-21 21:58:50,323 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Enma Solaire, de rang S et divinite / Enma
[2025-01-21 21:58:51,972 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bogos, de rang Boss
[2025-01-21 21:58:54,356 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Enma Solaire, de rang S et divinite / Enma
[2025-01-21 21:58:56,915 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Typ, de rang S spÃ©cial
[2025-01-21 21:58:59,397 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Inamygal, de rang Boss
[2025-01-21 21:59:01,799 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Typ, de rang S spÃ©cial
[2025-01-21 21:59:04,179 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu El Dragonor, de rang S et trÃ©sor
[2025-01-21 21:59:07,838 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-21 21:59:09,259 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-21 21:59:11,802 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Infiniris, de rang Boss
[2025-01-21 21:59:52,750 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Taprice, de rang Boss
[2025-01-21 21:59:53,895 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-21 21:59:55,744 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Grolos, de rang Boss
[2025-01-21 21:59:58,149 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Enma, de rang S et divinite / Enma
[2025-01-22 12:42:56,026 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 323, in reset
    a = "a" + 1
       ~~~~^~~
TypeError: can only concatenate str (not "int") to str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-01-22 12:42:55
- Commande qui a causé l'erreur : reset
[2025-01-22 19:18:28,953 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 415, in yokai
    if cooldown == True :
       ^^^^^^^^
UnboundLocalError: cannot access local variable 'cooldown' where it is not associated with a value


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-01-22 19:18:28
- Commande qui a causé l'erreur : reset
[2025-01-22 19:20:28,253 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu HÃ©morhino, de rang D
[2025-01-22 19:21:13,017 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Kuroi Jibanyan, de rang S spÃ©cial
[2025-01-22 19:21:31,127 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-22 19:21:31,618 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Balopendre, de rang S spÃ©cial
[2025-01-22 19:21:34,286 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Kuroi Jibanyan, de rang S spÃ©cial
[2025-01-22 19:21:36,697 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu ThÃ©opportun, de rang E
[2025-01-22 19:21:39,113 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Typ, de rang S spÃ©cial
[2025-01-22 19:21:41,506 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-22 19:21:43,995 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu MathiptÃ¨re, de rang S spÃ©cial
[2025-01-22 19:21:46,534 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu MathiptÃ¨re, de rang S spÃ©cial
[2025-01-22 19:21:49,058 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jibanyan (Lightside), de rang S spÃ©cial
[2025-01-22 19:21:51,513 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Kuroi Jibanyan, de rang S spÃ©cial
[2025-01-22 19:21:53,961 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Porcinator, de rang Boss
[2025-01-22 19:21:56,373 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Typ, de rang S spÃ©cial
[2025-01-22 19:21:58,744 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu LoupÃ©pin, de rang E
[2025-02-18 14:31:50,234 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 14:31:57,477 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 426, in yokai
    last_claim = int(brute_inventory[str(ctx.author.id)]["last_claim"])
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
KeyError: 'last_claim'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 14:31:56
- Commande qui a causé l'erreur : yokai
[2025-02-18 14:32:24,866 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu PiÃ¨tros, de rang E
[2025-02-18 14:33:34,323 - INFO]__main__: __hubble a utilisé le /reset dans random
[2025-02-18 14:33:42,011 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Volibrius, de rang C
[2025-02-18 15:24:35,674 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu DON-CHAN, de rang C
[2025-02-18 15:24:37,685 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Woko, de rang E
[2025-02-18 15:24:39,007 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Jalouseriz, de rang E
[2025-02-18 15:24:41,460 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bushidos, de rang A
[2025-02-18 15:24:44,296 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Fryzeur, de rang E
[2025-02-18 15:24:46,669 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Nihilistik, de rang E
[2025-02-18 15:24:49,203 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Oronyan, de rang S
[2025-02-18 15:24:51,732 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Rouminet, de rang C
[2025-02-18 15:24:53,437 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Whispavret, de rang C
[2025-02-18 15:24:56,084 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Poulpulaire, de rang C
[2025-02-18 15:24:58,615 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Verrascible, de rang B
[2025-02-18 15:25:00,792 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Fourchelangue, de rang C
[2025-02-18 15:25:03,569 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Nonjuan, de rang A
[2025-02-18 15:25:06,049 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Squarlett, de rang E
[2025-02-18 15:25:08,571 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Couchtar, de rang E
[2025-02-18 15:25:11,541 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Roupilion, de rang E
[2025-02-18 15:25:14,180 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Maudieuse, de rang E
[2025-02-18 15:25:15,898 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu DÃ©potache, de rang B
[2025-02-18 15:25:19,007 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Big O'Max, de rang D
[2025-02-18 15:25:21,778 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu DÃ©shirley, de rang E
[2025-02-18 15:25:23,693 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Tapur, de rang C
[2025-02-18 15:25:26,681 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Originyan, de rang E
[2025-02-18 15:25:28,998 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Pandore, de rang S et trÃ©sor
[2025-02-18 15:25:30,724 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Potache, de rang E
[2025-02-18 15:25:34,006 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Zazel le Bon, de rang S et divinite / Enma
[2025-02-18 15:25:36,840 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Pikor, de rang D
[2025-02-18 15:25:39,411 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu CrÃ¢nÃ©idoscope, de rang C
[2025-02-18 15:25:41,113 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Margoth, de rang C
[2025-02-18 15:25:43,971 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Curryeux, de rang E
[2025-02-18 15:25:46,599 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Originyan, de rang E
[2025-02-18 15:25:47,054 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 509, in yokai
    brute_inventory[str(ctx.author.id)][Yokai_choice]["count"] += 1
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
TypeError: list indices must be integers or slices, not str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 15:25:46
- Commande qui a causé l'erreur : yokai
[2025-02-18 15:29:24,683 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Chihuaglagla, de rang E
[2025-02-18 15:29:26,373 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Crampaud, de rang E
[2025-02-18 15:29:27,838 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Aramis Taire, de rang B
[2025-02-18 15:29:30,298 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Tapur, de rang C
[2025-02-18 15:29:30,798 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 509, in yokai
    brute_inventory[str(ctx.author.id)][Yokai_choice][1] += 1
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 15:29:30
- Commande qui a causé l'erreur : yokai
[2025-02-18 15:30:49,250 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Outransss, de rang E
[2025-02-18 15:30:50,686 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ã‰gare-dare, de rang E
[2025-02-18 15:30:51,887 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2025-02-18 15:30:54,597 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Kapunki, de rang A
[2025-02-18 15:30:57,518 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Herbert, de rang E
[2025-02-18 15:31:00,044 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Mythovni, de rang C
[2025-02-18 15:31:02,614 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Mochimacho, de rang D
[2025-02-18 15:31:05,007 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Felipaix, de rang E
[2025-02-18 15:31:07,623 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ultramax K, de rang B
[2025-02-18 15:31:10,294 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bradguette, de rang E
[2025-02-18 15:31:10,767 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 509, in yokai
    brute_inventory[str(ctx.author.id)][Yokai_choice][1] += 1
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 511, in yokai
    brute_inventory[str(ctx.author.id)][Yokai_choice][1] = 2
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list assignment index out of range


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 15:31:10
- Commande qui a causé l'erreur : yokai
[2025-02-18 15:32:50,550 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Marra, de rang D
[2025-02-18 15:32:52,077 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Charivariz, de rang D
[2025-02-18 15:32:53,458 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Erzestine, de rang D
[2025-02-18 15:32:56,268 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Grovide, de rang B
[2025-02-18 15:32:58,642 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu T. Energison, de rang S
[2025-02-18 15:33:01,275 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Kyubot, de rang S
[2025-02-18 15:33:03,748 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Dagamberge, de rang C
[2025-02-18 15:33:05,957 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Mochimacho, de rang D
[2025-02-18 15:33:06,406 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 514, in yokai
    yokai_embed.add_field(name=f"vous l'avez deja eu. Vous en avez donc {brute_inventory[str(ctx.author.id)][Yokai_choice]["count"]}", value="Faite `/inventaire` pour voir votre inventaire.")
                                                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
TypeError: list indices must be integers or slices, not str


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 15:33:05
- Commande qui a causé l'erreur : yokai
[2025-02-18 15:33:54,554 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu DÃ©ballerine, de rang S et legendaire
[2025-02-18 15:33:56,261 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Pachycoul, de rang D
[2025-02-18 15:33:57,731 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu PÃ©pÃ©sbrouf, de rang C
[2025-02-18 15:34:00,902 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Carquentin, de rang E
[2025-02-18 15:34:02,235 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Battant, de rang B
[2025-02-18 15:34:05,647 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Noripop, de rang E
[2025-02-18 15:34:08,108 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ralbouc, de rang C
[2025-02-18 15:34:10,731 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Originyan, de rang E
[2025-02-18 15:34:16,955 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:17:43,459 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:22:38,229 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:23:44,505 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:24:52,893 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:25:50,485 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:26:28,130 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:29:40,772 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:31:00,503 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:31:00,996 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 670, in inventaire
    inv_embed.set_footer(name="V1.5 Maintenant les Yo-kais se stackent !")
TypeError: Embed.set_footer() got an unexpected keyword argument 'name'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-18 16:31:00
- Commande qui a causé l'erreur : inventaire
[2025-02-18 16:31:50,239 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 16:42:25,072 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Marra, de rang D
[2025-02-18 16:42:29,421 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Bouh, de rang C
[2025-02-18 16:42:31,787 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu DÃ©mophage, de rang S
[2025-02-18 16:42:33,223 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Kyubot, de rang S
[2025-02-18 16:42:34,949 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Gloups, de rang D
[2025-02-18 16:42:38,013 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Whispavret, de rang C
[2025-02-18 16:42:40,096 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu MaÃ®tre Oden, de rang A
[2025-02-18 16:42:42,670 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Gambeth, de rang E
[2025-02-18 16:42:45,128 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Non-non, de rang D
[2025-02-18 16:42:47,616 - INFO]__main__: mart1max a utilisé le /yokai dans random, il a eu Papiltension, de rang D
[2025-02-18 16:43:28,040 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 17:26:40,518 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-02-18 17:28:21,401 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-18 17:28:26,281 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-02-22 12:33:44,445 - INFO]__main__: __hubble a utilisé le /inventaire dans random
[2025-02-22 13:10:09,661 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 425, in stats
    stats_embed = discord.Embed(color=discord.Color.green, title="Voici les stats de l'inventaire :")
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 179, in __init__
    self.colour = colour if colour is not None else color
    ^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 323, in colour
    raise TypeError(f'Expected discord.Colour, int, or None but received {value.__class__.__name__} instead.')
TypeError: Expected discord.Colour, int, or None but received method instead.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-22 13:10:09
- Commande qui a causé l'erreur : stats
[2025-02-22 13:11:02,852 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 425, in stats
    stats_embed = discord.Embed(color=discord.Colour.green, title="Voici les stats de l'inventaire :")
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 179, in __init__
    self.colour = colour if colour is not None else color
    ^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 323, in colour
    raise TypeError(f'Expected discord.Colour, int, or None but received {value.__class__.__name__} instead.')
TypeError: Expected discord.Colour, int, or None but received method instead.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-22 13:11:02
- Commande qui a causé l'erreur : stats
[2025-02-22 13:12:19,251 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-02-22 13:15:03,685 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-02-22 13:16:59,991 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-02-22 13:18:59,724 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-02-28 11:47:37,397 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 747, in inventaire
    error_embed = discord.Embed(
                  ^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 179, in __init__
    self.colour = colour if colour is not None else color
    ^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 323, in colour
    raise TypeError(f'Expected discord.Colour, int, or None but received {value.__class__.__name__} instead.')
TypeError: Expected discord.Colour, int, or None but received method instead.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 11:47:36
- Commande qui a causé l'erreur : inventaire
[2025-02-28 11:48:49,282 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-02-28 11:52:05,402 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Papa Ress, de rang E
[2025-02-28 19:44:59,109 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Noko, de rang E
[2025-02-28 19:44:59,535 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 601, in yokai
    await save_inv(brute_inventory)
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 66, in save_inv
    with open(f"./files/inventory/{str(id)}.json", "w", encoding="utf-8") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './files/inventory/<built-in function id>.json'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 19:44:59
- Commande qui a causé l'erreur : yokai
[2025-02-28 19:46:36,453 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 484, in yokai
    brute_inventory = await get_inv(ctx.author.id)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 57, in get_inv
    with open(f"./files/inventory/{str(id)}.json") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './files/inventory/882241177578004542.json'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 19:46:35
- Commande qui a causé l'erreur : yokai
[2025-02-28 19:47:22,205 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Humidon, de rang E
[2025-02-28 19:47:22,605 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 601, in yokai
    await save_inv(brute_inventory)
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 66, in save_inv
    with open(f"./files/inventory/{str(id)}.json", "w", encoding="utf-8") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 22] Invalid argument: './files/inventory/<built-in function id>.json'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 19:47:22
- Commande qui a causé l'erreur : yokai
[2025-02-28 20:10:10,081 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Darabajoie, de rang E
[2025-02-28 20:14:38,714 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Loubarbare, de rang E
[2025-02-28 20:14:40,002 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Duc Delafeuille, de rang E
[2025-02-28 20:14:41,712 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Maudieuse, de rang E
[2025-02-28 20:14:44,119 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ange Garudien, de rang S
[2025-02-28 20:14:46,649 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Chihuaglagla, de rang E
[2025-02-28 20:14:49,059 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Woko, de rang E
[2025-02-28 20:14:51,451 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Lama Laya, de rang C
[2025-02-28 20:15:01,229 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bouh, de rang C
[2025-02-28 20:15:01,859 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Feuildechou, de rang A
[2025-02-28 20:15:04,346 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Ã‰garoni, de rang E
[2025-02-28 20:15:06,790 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Felipaix, de rang E
[2025-02-28 20:15:09,993 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Moqua glacÃ©, de rang B
[2025-02-28 20:15:11,550 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Outransss, de rang E
[2025-02-28 20:15:13,987 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Chiperpiou, de rang E
[2025-02-28 20:15:16,349 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu YokÅ“il, de rang D
[2025-02-28 20:15:19,279 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Loubarbare, de rang E
[2025-02-28 20:19:10,634 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-02-28 20:19:14,506 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-02-28 20:19:16,448 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-02-28 20:27:10,447 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 362, in reset
    brute_inventory = await get_data(input_id)
                            ^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 74, in get_data
    with open(path) as f:
         ^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '882241177578004542'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 20:27:09
- Commande qui a causé l'erreur : reset
[2025-02-28 20:33:54,594 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 372, in reset
    await save_data ()
          ^^^^^^^^^^^^
TypeError: save_data() missing 2 required positional arguments: 'path' and 'data'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-02-28 20:33:54
- Commande qui a causé l'erreur : reset
[2025-02-28 20:35:33,201 - WARNING]__main__: __hubble a utilisé le /reset dans random
[2025-02-28 20:35:48,602 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-02-28 20:35:52,840 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu PersÃ©vÃ©fant, de rang C
[2025-02-28 20:46:41,680 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-02-28 20:47:06,349 - INFO]__main__: original_villager a utilisé le /yokai dans random, il a eu Felipaix, de rang E
[2025-02-28 20:47:23,366 - WARNING]__main__: __hubble a utilisé le /stats dans random, sur l'input inventory
[2025-03-01 10:15:23,267 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-03-01 16:05:02,857 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-01 16:05:06,940 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Espi, de rang B
[2025-03-01 16:05:33,911 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-01 16:56:36,865 - INFO]__main__: original_villager a utilisé le /yokai dans random, il a eu Lama Laya, de rang C
[2025-03-01 16:58:40,126 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 793, in trade
    ask_embed = discord.Embed(color=discord.Color.green,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 179, in __init__
    self.colour = colour if colour is not None else color
    ^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\embeds.py", line 323, in colour
    raise TypeError(f'Expected discord.Colour, int, or None but received {value.__class__.__name__} instead.')
TypeError: Expected discord.Colour, int, or None but received method instead.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-01 16:58:39
- Commande qui a causé l'erreur : trade
[2025-03-01 16:59:41,402 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 801, in trade
    value=f"Le Yo-kai **{son_yokai}** de rang **{classid_to_class(author_inv[son_yokai][0])}**"
                                                                  ~~~~~~~~~~^^^^^^^^^^^
KeyError: 'Lama Laya'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-01 16:59:40
- Commande qui a causé l'erreur : trade
[2025-03-02 15:00:19,052 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-02 15:10:41,160 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu GranpapÃ©ti, de rang E
[2025-03-02 15:13:08,704 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 484, in yokai
    brute_inventory = await get_inv(ctx.author.id)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 58, in get_inv
    data = json.load(f)
           ^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-02 15:13:08
- Commande qui a causé l'erreur : yokai
[2025-03-02 15:13:28,506 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Potache, de rang E
[2025-03-02 15:13:34,064 - INFO]__main__: original_villager a utilisé le /yokai dans random, il a eu Onisoi, de rang D
[2025-03-02 15:14:30,781 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-02 15:36:45,736 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-03-02 15:36:56,032 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-03-02 15:37:27,881 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-03-02 15:38:09,396 - INFO]__main__: __hubble a utilisé le /help dans random
[2025-03-03 20:15:57,676 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-03 20:16:03,334 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-03 20:18:09,007 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-03 20:18:23,532 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-03 20:19:42,300 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 11:59:50,945 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 11:59:59,001 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 12:11:39,226 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Darabajoie, de rang E
[2025-03-05 12:12:48,650 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 826, in trade
    value= f"Le Yo-kai **{ton_yokai}** de rang **{classid_to_class(author_inv[ton_yokai][0])}**"
                                                                   ~~~~~~~~~~^^^^^^^^^^^
KeyError: ('Potache', 'Darabajoie')


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-05 12:12:48
- Commande qui a causé l'erreur : trade
[2025-03-05 12:13:38,711 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 826, in trade
    value= f"Le Yo-kai **{ton_yokai}** de rang **{classid_to_class(author_inv[ton_yokai][0])}**"
                                                                   ~~~~~~~~~~^^^^^^^^^^^
KeyError: ('Potache', 'Darabajoie')


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-05 12:13:38
- Commande qui a causé l'erreur : trade
[2025-03-05 12:14:17,023 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Onisoi',) contre ('Potache', 'Darabajoie'), dans random
[2025-03-05 12:14:29,503 - INFO]__main__: original_villager a accepté le trade de __hubble, il demandait ('Onisoi',) contre ('Potache', 'Darabajoie'), dans random
[2025-03-05 12:14:37,928 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 12:14:45,527 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 12:40:38,633 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 827, in trade
    offered_yokai += f"> Le Yo-kai **{asked_yokai}** de rang **{classid_to_class(author_inv[asked_yokai][0])}**\n "
                                                                                 ~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'Potache'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-05 12:40:38
- Commande qui a causé l'erreur : trade
[2025-03-05 12:41:25,482 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Potache', 'Darabajoie') contre ('Onisoi',), dans random
[2025-03-05 12:42:54,947 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Potache', 'Darabajoie') contre ('Onisoi',), dans random
[2025-03-05 12:43:01,581 - INFO]__main__: original_villager a accepté le trade de __hubble, il demandait ('Potache', 'Darabajoie') contre ('Onisoi',), dans random
[2025-03-05 12:43:51,319 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Onisoi',) contre ('Potache', 'Darabajoie'), dans random
[2025-03-05 12:43:54,332 - INFO]__main__: original_villager a accepté le trade de __hubble, il demandait ('Onisoi',) contre ('Potache', 'Darabajoie'), dans random
[2025-03-05 12:50:47,640 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Bicorniot, de rang B
[2025-03-05 12:51:25,461 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 12:52:05,135 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu PadrÃ©zo, de rang D
[2025-03-05 12:52:05,531 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 607, in yokai
    yokai_embed.set_footer(name="V2.5 : Maintenant vous pouvez trades des yokai ('/help User' pour plus d'infos dans la categorie .trade)")
TypeError: Embed.set_footer() got an unexpected keyword argument 'name'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-05 12:52:05
- Commande qui a causé l'erreur : yokai
[2025-03-05 12:54:27,175 - INFO]__main__: __hubble a utilisé le /yokai dans random, il a eu Gastong, de rang E
[2025-03-05 17:07:13,841 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 17:07:25,920 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 17:09:18,439 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Potache',) contre ('Gastong',), dans random
[2025-03-05 17:09:20,399 - INFO]__main__: original_villager a accepté le trade de __hubble, il demandait ('Potache',) contre ('Gastong',), dans random
[2025-03-05 17:09:41,109 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 17:09:51,716 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 17:13:00,326 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-05 17:13:26,476 - INFO]__main__: __hubble a demandé un trade à original_villager, il demande ('Gastong',) contre ('Bicorniot',), dans random
[2025-03-05 17:13:29,209 - INFO]__main__: original_villager a accepté le trade de __hubble, il demandait ('Gastong',) contre ('Bicorniot',), dans random
[2025-03-05 17:13:41,907 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:15:09,018 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:15:09,604 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:15:09
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:16:40,126 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:16:40,747 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:16:40
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:17:29,064 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:17:29,625 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:17:29
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:18:19,532 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:19:27,226 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:19:28,070 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:19:27
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:20:47,294 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:20:47,810 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:20:47
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:22:26,232 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:22:26,752 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 716, in medallium
    return await send_embed(ctx, inv_embed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 135, in send_embed
    await ctx.send(embed=embed)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 223, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In data.embeds.0.fields.0.value: Must be 1024 or fewer in length.


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-14 19:22:26
- Commande qui a causé l'erreur : medallium
[2025-03-14 19:30:39,023 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:31:43,402 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-14 19:33:52,284 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 16:52:19,932 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 16:52:20,328 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 722, in medallium
    await ctx.send(view=dropdown)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\ext\commands\context.py", line 1077, in send
    await self.interaction.response.send_message(**kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\interactions.py", line 839, in send_message
    params = interaction_message_response_params(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\webhook\async_.py", line 581, in interaction_message_response_params
    data['components'] = view.to_components()
                         ^^^^^^^^^^^^^^^^^^^^
TypeError: View.to_components() missing 1 required positional argument: 'self'


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 16:52:19
- Commande qui a causé l'erreur : medallium
[2025-03-16 16:54:49,740 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 16:54:50,194 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 722, in medallium
    await ctx.send(view=dropdown())
                        ^^^^^^^^^^
TypeError: 'Inv_dropdown_view' object is not callable


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 16:54:49
- Commande qui a causé l'erreur : medallium
[2025-03-16 16:56:46,004 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 16:56:46,411 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 722, in medallium
    await ctx.send("test", view=dropdown())
                                ^^^^^^^^^^
TypeError: 'Inv_dropdown_view' object is not callable


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 16:56:46
- Commande qui a causé l'erreur : medallium
[2025-03-16 20:24:47,147 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:24:47,584 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 722, in medallium
    await ctx.send("test", view=dropdown())
                                ^^^^^^^^^^
TypeError: 'Inv_dropdown_view' object is not callable


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 20:24:47
- Commande qui a causé l'erreur : medallium
[2025-03-16 20:26:27,523 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:36:00,157 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:52:51,495 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:56:02,148 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:58:04,007 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:58:04,875 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 770, in medallium
    classes_name = next(iter(yokai_list_brute))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 20:58:04
- Commande qui a causé l'erreur : medallium
[2025-03-16 20:59:18,078 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 20:59:18,790 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "d:\User\lubin\Documents\#Programmation\python\bot discord\yokai bot\script\main.py", line 770, in medallium
    classes_name = next(iter(yokai_list_brute))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 20:59:18
- Commande qui a causé l'erreur : medallium
[2025-03-16 23:16:58,820 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 23:16:59,547 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "c:\Users\lubintschirhart\Desktop\bot discord\yokai bot\script\main.py", line 804, in medallium
    classes_name = next(iter(yokai_list_brute))
StopIteration


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 23:16:59
- Commande qui a causé l'erreur : medallium
[2025-03-16 23:19:35,556 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 23:19:36,387 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "c:\Users\lubintschirhart\Desktop\bot discord\yokai bot\script\main.py", line 804, in medallium
    classes_name = next(iter(yokai_list_brute))
StopIteration


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 23:19:35
- Commande qui a causé l'erreur : medallium
[2025-03-16 23:24:03,702 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-16 23:24:04,356 - ERROR]__main__: 
Error message :
Traceback (most recent call last):
  File "c:\Users\lubintschirhart\Desktop\bot discord\yokai bot\script\main.py", line 809, in medallium
    classes_name = next(iter(yokai_list_brute))
StopIteration


- Nom d'utilisateur / id : __hubble/882241177578004542
- Heure de l'erreur : 2025-03-16 23:24:03
- Commande qui a causé l'erreur : medallium
[2025-03-17 08:13:52,150 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 08:15:32,931 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 08:20:22,588 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 08:23:17,241 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 20:36:02,219 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 20:36:54,452 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 20:39:58,191 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-17 20:41:46,168 - INFO]__main__: __hubble a utilisé le /medallium dans random
[2025-03-18 19:29:20,748 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-18 19:29:53,546 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-18 20:55:34,074 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-18 20:56:55,633 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-18 20:59:48,433 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:13:23,288 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:14:14,216 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:16:00,633 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:16:51,536 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:20:26,293 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:23:21,979 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:26:37,889 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:28:12,002 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:29:20,796 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 15:53:26,624 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:23:29,462 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:25:30,577 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:28:13,343 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:29:25,692 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:30:04,722 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
[2025-03-19 17:47:39,569 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 17:51:51,677 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 17:52:12,492 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 17:53:10,269 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 17:53:34,533 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 17:53:50,873 - INFO]__main__: __hubble a utilisÃ© le /help dans random
[2025-03-19 18:20:43,260 - INFO]__main__: __hubble a utilisÃ© le /medallium dans random
